store/storetest/team_store.go

```diff
@@ -399,7 +405,11 @@ func testTeamStoreGetByInviteId(t *testing.T, ss store.Store) {
 	o2.Email = MakeEmail()
 	o2.Type = model.TEAM_OPEN
 
-	if r1 := <-ss.Team().GetByInviteId(save1.Data.(*model.Team).InviteId); r1.Err != nil {
+	if _, err := ss.Team().Save(&o2); err != nil {
```

Comment:
This `Save` wasn't there before.

Reply:
thanks will be more careful with merging next time...